OpenEdXMobile/src/main/java/org/edx/mobile/view/CourseTabsDashboardActivity.kt

```diff
@@ -0,0 +1,100 @@
+package org.edx.mobile.view
+
+import android.content.Context
+import android.content.Intent
+import android.os.Bundle
+import dagger.hilt.android.AndroidEntryPoint
+import org.edx.mobile.R
+import org.edx.mobile.base.BaseFragmentActivity
+import org.edx.mobile.deeplink.ScreenDef
+import org.edx.mobile.event.CourseDashboardRefreshEvent
+import org.edx.mobile.event.CourseUpgradedEvent
+import org.edx.mobile.http.notifications.SnackbarErrorNotification
+import org.edx.mobile.interfaces.RefreshListener
+import org.edx.mobile.interfaces.SnackbarStatusListener
+import org.edx.mobile.model.api.EnrolledCoursesResponse
+import org.edx.mobile.util.NetworkUtil
+import org.greenrobot.eventbus.EventBus
+
+@AndroidEntryPoint
+class CourseTabsDashboardActivity : BaseFragmentActivity(), SnackbarStatusListener,
+    RefreshListener {
+
+    private lateinit var snackbarErrorNotification: SnackbarErrorNotification
```

Comment:
rename it to `snackBarErrorNotification`.

Reply:
Should I rename the file as well?

Reply:
let's remain the same name currently used in the code.